Makefile

```diff
@@ -3,6 +3,17 @@ IMG ?= quay.io/ecosystem-appeng/dbaas-e2e-test-harness:latest
 DIR := $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
 OUT_FILE := "$(DIR)dbaas-e2e-test-harness"
 
+install-tools:
+	go install golang.org/x/tools/cmd/goimports@latest
+	go install github.com/mgechev/revive
+
+fmt: install-tools
+	goimports -w .
+
+lint: install-tools
+	goimports -d .
+	revive -config $(DIR)config.toml ./...
+
 build:
```

Comment:
Any reason not to have build depend on fmt + lint?

Reply:
This is because gofmt is very limited and its functionalities are included in goimports. golint has been deprecated with revive as its replacement.